src/app/[lng]/contact/page.tsx

```diff
@@ -1,5 +1,115 @@
-import React from 'react';
+"use client"
 
-const Datasets: React.FC = () => <div>page</div>;
+import { TextField, TextAreaField, Flex, Button, SelectField, Heading } from '@aws-amplify/ui-react';
+import React, { useRef, FormEvent } from 'react';
+import styled from 'styled-components';
 
-export default Datasets;
+const ContactForm = () => {
+
+
+    const firstNameRef = useRef<HTMLInputElement>(null);
+    const lastNameRef = useRef<HTMLInputElement>(null);
+    const emailRef = useRef<HTMLInputElement>(null);
+    const phoneNumberRef = useRef<HTMLInputElement>(null); 
+    const topicRef = useRef<HTMLSelectElement>(null);
+    const messageRef = useRef<HTMLTextAreaElement>(null);
+
+    const handleSubmit = async (e: FormEvent) => {
+        e.preventDefault();
+
+
+        const formData = {
+            firstName: firstNameRef.current?.value,
+            lastName: lastNameRef.current?.value,
+            email: emailRef.current?.value,
+            phoneNumber: phoneNumberRef.current ? phoneNumberRef.current.value : '',
+            topic: topicRef.current?.value,
+            message: messageRef.current?.value
+        };
+
+        try {
+            const response = await fetch('/api/sendEmail', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify(formData)
+            });
+
+            if (response.ok) {
+                const result = await response.json();
+                console.log('Email sent successfully:', result);
+
+
+                // Handle successful email sending (e.g., showing a success message)
+            } else {
+                throw new Error('Failed to send email');
+            }
+        } catch (error) {
+            console.error('Error sending email:', error);
+            // Handle errors (e.g., showing an error message)
+        }
+
+        (e.target as HTMLFormElement).reset();
+    };
+
+
+    const StyledForm = styled(Flex)`
+    max-width: 40%; 
+    margin: 0 auto; 
+    padding: 20px;
+    flex-direction: column;
+   
+    
+  
+`;
+
+    const StyledInput = styled(TextField)`
+    margin-bottom: 16px; 
+`;
+
+    const StyledTextArea = styled(TextAreaField)`
+
+`;
+
+    const StyledSelect = styled(SelectField)`
+    margin-bottom: 16px; 
+`;
+
+    const StyledButton = styled(Button)`
```

Comment:
I don't think we will need styled components for all these elements. Buttons and text fields will likely be consistent across the app. If we need to modify the styles for those we should do it in the theme or pull out into a reusable component